src/routes/AppRouter.js

```diff
@@ -2,11 +2,13 @@ import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 import LandingPage from '../pages/loginPage';
 import Homepage from '../pages/hompage';
+import Card from '../components/registration/index';
```

Comment:
you might want to remove index from the end of this so you can have something like `import Card from '../components/registration` instead